package.json

```diff
@@ -21,19 +24,40 @@
   "homepage": "https://github.com/gektin/max-descriptions#readme",
   "dependencies": {
     "axios": "^0.20.0",
-    "bluebird": "^3.7.2",
     "express": "^4.17.1",
     "morgan": "^1.10.0",
     "mysql": "^2.18.1",
     "mysql2": "^2.2.5",
-    "nodemon": "^2.0.4",
-    "sequelize": "^6.3.5",
-    "supertest": "^5.0.0"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.2.0",
+    "sequelize": "^6.3.5"
   },
   "devDependencies": {
-    "@babel/cli": "^7.11.6",
```

Comment:
I think it's interesting that your removed `@babel/cli` . Would like to talk about this for my own benefit.